modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/beans/Container.java

```diff
@@ -25,7 +25,7 @@
     private String imageId;
     private int replicaNumber;
     private String serviceUrl;
-    private Map<String,String> envVariables;
+    private List envVariables;
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/7d70ba5f5bbacfe7789db67072db49c2ff893b78